content/en/docs/reference/config/security/conditions/index.md

```diff
@@ -7,7 +7,7 @@ aliases:
 ---
 
 This page describes the supported keys and value formats you can use as conditions
-in the `when` field of [authorization policy resources](/docs/reference/config/security/authorization-policy/).
+in the `when` field of [authorization policy resources](/docs/reference/config/security/authorization-policy/#Condition).
```

Comment:
I think this is equally confusing because, as currently worded, the link is to the "authorization policy resource", not a "condition resource". Maybe something like this would be more clear?
```suggestion
in the `when` field of an [authorization policy rule](/docs/reference/config/security/authorization-policy/#Rule).
```